packages/p2p/src/components/composite-calendar/composite-calendar-mobile/composite-calendar-mobile.tsx

```diff
@@ -151,16 +172,18 @@ const CompositeCalendarMobile = ({
                         <div className='composite-calendar-mobile__custom-date-range'>
                             <DatePicker
                                 className='composite-calendar-mobile__custom-date-range-start-date'
+                                disabled={selected_date_range?.value !== CUSTOM_KEY}
```

Comment:
can we store this check into a const value? something like:

`const has_selected_date_range = selected_date_range?.value === CUSTOM_KEY`

then just pass it accordingly to the other props:

```suggestion
                                disabled={!has_selected_date_range}
```